hypha/apply/activity/models.py

```diff
@@ -136,11 +136,11 @@ def __str__(self):
     @classmethod
     def visibility_for(cls, user):
         if user.is_apply_staff:
-            return [APPLICANT, TEAM, REVIEWER, PARTNER, ALL]
+            return getattr(settings, "COMMENT_VISIBILITY_ADMIN", [])
```

Comment:
Can we not use `return settings.COMMENT_VISIBILITY_ADMIN` etc. here? It will always be set since it is in base.py.

Reply:
Also prefer `COMMENT_VISIBILITY_STAFF` instead of `COMMENT_VISIBILITY_ADMIN`.